### PR TITLE
Update ExtendedBeanSerializer docs

### PR DESCRIPTION
**Pull request by tomforster**
Added note on the columnAnnotation=true configuration parameter requirement. Related to #1868

### CODE REVIEW

File: querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
 /**
  * {@code ExtendedBeanSerializer} outputs primary key based {@code equals}, {@code hashCode} and
  * {@code toString} implementations
+ * 
+ * Requires column annotation generation to be enabled
```
Comment (Shredder121) [inline querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java]: I'll wait for @timowest or @johnktims to chime in for the text.
But there is a trailing space at line 31.

**Comment by Shredder121** [comment]
Yes, I think the note is useful, it should fix future problems.

**Comment by tomforster** [comment]
Removed the trailing space.

**Comment by Shredder121** [comment]
Sorry, we're having some issues with Travis.
Could you maybe squash the commits, and add the text `[skip ci]` to the commit message?

**Comment by tomforster** [comment]
Done.

**Comment by Shredder121** [comment]
You can also have the commit message like this:
```
Added note on required columnAnnotations for ExtendedBeanSerializer

[skip ci] 
```
Sorry for not making that clear

**Comment by tomforster** [comment]
Ok, have changed that.

**Comment by robertandrewbain** [comment]
Can we fetch, rebase and remove the `[skip ci] ` now, as #1874 has resolved the CI issues we were experiencing.

**Comment by robertandrewbain** [comment]
Rebased against master and removed the [skip ci] - will be closed by https://github.com/querydsl/querydsl/pull/1873

**Comment by Shredder121** [comment]
@tomforster Thanks for the input, much appreciated!

**Pull request closed without merging**